resources/public/api/conf/0.0/modules/subscriptions.raml

```diff
@@ -30,6 +30,31 @@ types:
       Represents a request to be subscribed to a Service
     type: object
     properties:
+      service:
+        description: The service to subscribe to.
+        type: HmrcService
+      agentCode:
+        description: The agent code
+        type: string
+      telephone:
+        description: The agency's telephone
+        type: string
+      email:
+        description: The agency's e-mail
+        type: string
+      address:
+        description: The agency's address
+        type: Address
+      utr:
+        description:  The agency's UTR
+        type: string
+
+  SubscriptionInfo:
+    displayName: Subscription Request
```

Comment:
displayName and description for SubscriptionInfo are the same as for SubscriptionRequest but should presumably be different?

Also the API platform expects JSON Schema not RAML types like we have here (and in agent-client-authorisation). Can merge this and change to JSON Schema later if you like though?